### PR TITLE
Events

### PR DESCRIPTION
**Pull request by itexpert120**
Resolves https://github.com/NEARBuilders/gateway/issues/126

### CODE REVIEW

**Comment by elliotBraem** [comment]
@itexpert120 Looks great, it works very well and I'm glad to see all the filters working. Some feedback:

- [ ] clear filters doesn't clear the date fields (I'm a bit confused by the props.setFilters being passed around from modals -- any way you think to simplify it all? Or reasoning to justify current setup?) 
- [x] I renamed "fetchEvents" to "fetchThings", as it wasn't really specific to events (also renamed to lib.everything-sdk)
- [ ] rename "test" to "every" once ready
- [ ] Imagine if we want others to use this calendar component, how can we have the theme all define in one place for easy customization?

**Comment by itexpert120** [comment]
> @itexpert120 Looks great, it works very well and I'm glad to see all the filters working. Some feedback:
> 
> * [ ]  clear filters doesn't clear the date fields (I'm a bit confused by the props.setFilters being passed around from modals -- any way you think to simplify it all? Or reasoning to justify current setup?)
> * [x]  I renamed "fetchEvents" to "fetchThings", as it wasn't really specific to events (also renamed to lib.everything-sdk)
> * [ ]  rename "test" to "every" once ready
> * [ ]  Imagine if we want others to use this calendar component, how can we have the theme all define in one place for easy customization?

Thanks for the review @elliotBraem. The reason for having setFilter passed to filter Modal is because the filters are stored in parent component. This makes handling filters easier. If you have a better solution, please share and I'll implement it.

File: apps/builddao/widget/lib/everything-sdk.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  }
+
+  const flattenedKeys = flattenObject(rawKeys, app, type);
+
+  let events = [];
+  flattenedKeys.forEach((key) =>
+    events.push(JSON.parse(Social.get(key, "final")))
+  );
+
+  return events;
```
Comment (elliotBraem) [inline apps/builddao/widget/lib/everything-sdk.jsx:35]: rename all "events" to "things"

Comment (itexpert120) [reply apps/builddao/widget/lib/everything-sdk.jsx:35]: kk

**Pull request merged**